med-web-app-front/src/components/attachments/upload-attachments.component.js

```diff
@@ -102,16 +144,80 @@ class UploadAttachmentsComponent extends Component {
             selectedFiles: undefined,
             progressInfos: [],
             message: [],
-
             fileInfos: [],
+            dragEnter: false,
         };
     }
 
-    selectFiles(event) {
+    deleteElement(index) {
+        let files = [...this.state.selectedFiles];
+        files.splice(index, 1);
+        if(files.length === 0) files = undefined
+        this.setState({
+            selectedFiles: files,
+        });
+    }
+
+    selectFiles() {
+        this.fileInput.current.click();
+    }
+
+    handleFiles(e) {
+        let selectedFiles = []
+        let files = [...e.target.files]
+        let errorMessage = []
+        if(this.state.selectedFiles !== undefined) selectedFiles = [...this.state.selectedFiles]
```

Comment:
```suggestion
        if (this.state.selectedFiles !== undefined) selectedFiles = [...this.state.selectedFiles]
```
И ниже. Это же JavaScript, к нему можно [ESLint](https://eslint.org/) прикрутить